pkg/utils/kube/resource_ops.go

```diff
@@ -242,13 +243,13 @@ func (k *kubectlResourceOperations) ApplyResource(ctx context.Context, obj *unst
 }
 
 func (k *kubectlResourceOperations) newApplyOptions(ioStreams genericclioptions.IOStreams, obj *unstructured.Unstructured, fileName string, validate bool, force bool, dryRunStrategy cmdutil.DryRunStrategy) (*apply.ApplyOptions, error) {
-	o := apply.NewApplyOptions(ioStreams)
+	o := &apply.ApplyOptions{}
```

Comment:
I was trying to run Argo CD e2e with the updated engine and found that a few `ApplyOptions` are missing. Since I already had test environment setup I've added missing fields and created PR to your fork: https://github.com/tommyknows/gitops-engine/pull/1